packages/common/src/ui/components/modals/ConfirmationModal/ConfirmationModal.stories.tsx

```diff
@@ -51,5 +52,16 @@ const Loading: Story = () => {
   );
 };
 
+const UseConfirmationModalHook: Story = () => {
+  const getConfirmation = useConfirmationModal({
+    title: 'Are you sure?',
+    message: 'This will delete all your data.',
+    onConfirm: () => alert('confirmed!'),
+  });
+
+  return <BaseButton onClick={getConfirmation}>Open Modal</BaseButton>;
```

Comment:
should I also put a story in the `hooks` section 🤔

Reply:
not essential, but it is a hook - so probably yes.